test/Sign.Cli.Test/AzureCredentialOptionsTests.cs

```diff
@@ -164,7 +164,6 @@ public void CreateDefaultAzureCredentialOptions_WhenNoOptionsAreSpecified_Exclud
 
             Assert.True(credentialOptions.ExcludeInteractiveBrowserCredential);
             Assert.True(credentialOptions.ExcludeSharedTokenCacheCredential);
-            Assert.True(credentialOptions.ExcludeVisualStudioCodeCredential);
```

Comment:
I removed this because it's now a compile-time error.

CS0618	'DefaultAzureCredentialOptions.ExcludeVisualStudioCodeCredential' is obsolete: 'VisualStudioCodeCredential is deprecated because the VS Code Azure Account extension on which this credential relies has been deprecated. Consider using other dev-time credentials, such as VisualStudioCredential, AzureCliCredential, AzureDeveloperCliCredential, AzurePowerShellCredential. See the Azure Account extension deprecation notice here: https://github.com/microsoft/vscode-azure-account/issues/964.'